how-to/software/upgrade-your-release.md

```diff
@@ -89,9 +98,9 @@ Configuration file '/etc/ssh/ssh_config'
 
 You should look at the differences between the files and decide what to do. The default response is to keep the current version of the file. There are situations where accepting the new version, like with `/boot/grub/menu.lst`, is required for the system to boot correctly with the new kernel.
 
-### Package removal
+### Removing Obsolete Packages
 
-After all packages are updated, you can choose to remove any obsolete, no-longer-needed packages:
+After all packages are updated, you can choose to remove any obsolete packages that are  no longer needed packages:
```

Comment:
Replace:

After all packages are updated, you can choose to remove any obsolete packages that are  no longer needed packages:

With:

After all packages are updated, you can choose to remove any obsolete packages that are  no longer needed: